core/domain/suggestion_services.py

```diff
@@ -606,11 +618,24 @@ def get_reviewable_suggestions(user_id, suggestion_type):
             user_id)
         language_codes = (
             contribution_rights.can_review_translation_for_language_codes)
-        all_suggestions = ([
-            get_suggestion_from_model(s) for s in (
+
+        in_review_translation_suggestions = []
+        if len(opportunity_summary_exp_ids) > 0:
+            in_review_translation_suggestions = (
                 suggestion_models.GeneralSuggestionModel
-                .get_in_review_translation_suggestions(
+            .get_in_review_translation_suggestions_with_exp_ids(
```

Comment:
Indent by 4, this is still within the scope of the paren two lines above.

Reply:
I will fix this.

Reply:
Done